### DISCUSSION TITLE
Welcome to INDIGO IAM Discussions!

### DISCUSSION THREAD

**Discussion by andreaceccanti** [indigo-iam/iam#370] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by norealroots** [comment]
Hello everyone!
My name is Tom, and I work at STFC-UKRI. I run and manage an instance of the IAM software for the [UK-IRIS](https://www.iris.ac.uk/) community. STFC also works closely alongside Andrea and the team where we can in order to support and facilitate the development of IAM, including technical effort.
Alongside Andrea I also led the set of the IAM User's Workshop, held in Jan 2021. The forum for this workshop and community can be found at: https://groups.google.com/g/indigo-iam-users
